src/main/java/Number.java

```diff
@@ -0,0 +1,30 @@
+import java.util.ArrayDeque;
+import java.util.Deque;
+
+public class Number {
+
+    private Deque<Integer> numbers;
+
+    public Number() {
+        this.numbers = new ArrayDeque<>();
+    }
+
+    public void add(Integer number) {
+        if (numbers.isEmpty()) {
+            numbers.add(number);
+        }
+    }
+
+    public Integer getFirstNumber() {
+        return numbers.pollFirst();
+    }
+
+    public int size() {
+        return numbers.size();
+    }
+    public boolean sameSize(int size) {
+        return this.size() == size;
+    }
+
+
```

Comment:
```suggestion
    }
    
    public boolean sameSize(int size) {
        return this.size() == size;
    }

```

개행도 클린코드를 위해 의미를 나타내긴 합니다. 개행도 신경써주세요.

Reply:
넵